<?xml version='1.0' encoding='UTF-8'?>
<root>
 <tabbed_widget name="Main Window" parent="main_window">
  <plotmatrix columns="1" tab_name="geometric" rows="2">
   <plot row="0" col="0">
    <range bottom="-0.013623" top="0.057492" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BI/x" custom_transform="noTransform" R="0" G="7"/>
    <curve B="17" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BI/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="17" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BI/z" custom_transform="noTransform" R="0" G="131"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range bottom="-0.065082" top="0.044979" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BP/x" custom_transform="noTransform" R="2" G="0"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BP/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="2" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BP/z" custom_transform="noTransform" R="0" G="150"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix columns="1" tab_name="inertia" rows="3">
   <plot row="0" col="0">
    <range bottom="-0.016605" top="0.029880" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BM/x" custom_transform="noTransform" R="0" G="0"/>
    <curve B="3" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BM/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="4" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BM/z" custom_transform="noTransform" R="0" G="138"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range bottom="0.000513" top="0.001161" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/I/x" custom_transform="noTransform" R="0" G="0"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/I/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/I/z" custom_transform="noTransform" R="3" G="90"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="2" col="0">
    <range bottom="0.230789" top="0.440534" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/m" custom_transform="noTransform" R="0" G="0"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix columns="1" tab_name="trajectory" rows="4">
   <plot row="0" col="0">
    <range bottom="-0.434794" top="1.481370" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/r/x" custom_transform="noTransform" R="0" G="0"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r/y" custom_transform="noTransform" R="255" G="3"/>
    <curve B="7" name="/quadrotor/geom_inertia_estimator/param_estimates/r/z" custom_transform="noTransform" R="0" G="121"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range bottom="-0.466662" top="1.035622" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/q/w" custom_transform="noTransform" R="0" G="0"/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/q/x" custom_transform="noTransform" R="5" G="0"/>
    <curve B="15" name="/quadrotor/geom_inertia_estimator/param_estimates/q/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/q/z" custom_transform="noTransform" R="0" G="128"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="2" col="0">
    <range bottom="-1.677202" top="1.822567" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/v/x" custom_transform="noTransform" R="0" G="16"/>
    <curve B="19" name="/quadrotor/geom_inertia_estimator/param_estimates/v/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="5" name="/quadrotor/geom_inertia_estimator/param_estimates/v/z" custom_transform="noTransform" R="0" G="128"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="3" col="0">
    <range bottom="-8.172533" top="7.710005" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="237" name="/quadrotor/geom_inertia_estimator/param_estimates/Omega/x" custom_transform="noTransform" R="0" G="10"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/Omega/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="16" name="/quadrotor/geom_inertia_estimator/param_estimates/Omega/z" custom_transform="noTransform" R="0" G="128"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix columns="1" tab_name="biases" rows="2">
   <plot row="0" col="0">
    <range bottom="-0.546396" top="1.518767" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/b_a/x" custom_transform="noTransform" R="0" G="0"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/b_a/y" custom_transform="noTransform" R="255" G="13"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/b_a/z" custom_transform="noTransform" R="0" G="128"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range bottom="-0.175482" top="0.150830" left="0.000000" right="24.991925"/>
    <limitY/>
    <curve B="255" name="/quadrotor/geom_inertia_estimator/param_estimates/b_Omega/x" custom_transform="noTransform" R="0" G="0"/>
    <curve B="7" name="/quadrotor/geom_inertia_estimator/param_estimates/b_Omega/y" custom_transform="noTransform" R="255" G="0"/>
    <curve B="0" name="/quadrotor/geom_inertia_estimator/param_estimates/b_Omega/z" custom_transform="noTransform" R="13" G="128"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <currentPlotMatrix index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <Plugins>
  <DataLoad_CSV>
   <default time_axis=""/>
  </DataLoad_CSV>
  <DataLoad_ROS_bags>
   <selected_topics list=""/>
  </DataLoad_ROS_bags>
  <DataLoad_ULog>
   <no_params/>
  </DataLoad_ULog>
  <ROS_Topic_Streamer>
   <selected_topics list="/quadrotor/geom_inertia_estimator/param_estimates"/>
  </ROS_Topic_Streamer>
  <RosoutPublisherROS/>
  <TopicPublisherROS/>
 </Plugins>
 <previouslyLoadedStreamer name="ROS_Topic_Streamer"/>
</root>

